sq/Makefile

```diff
@@ -37,24 +37,25 @@ install: build-release
 
 test-dsm:
 	cargo run -- --version
-	./tests/dsm/key_expiration.sh --rsa2k
-	./tests/dsm/key_expiration.sh --rsa3k
-	./tests/dsm/key_expiration.sh --rsa4k
-	./tests/dsm/key_expiration.sh --p256
-	./tests/dsm/key_expiration.sh --p384
-	./tests/dsm/key_expiration.sh --p521
-	./tests/dsm/key_expiration.sh --cv25519
-	./tests/dsm/extract_dsm_import_gpg_auto.tcl --rsa2k
-	./tests/dsm/extract_dsm_import_gpg_auto.tcl --rsa3k
-	./tests/dsm/extract_dsm_import_gpg_auto.tcl --rsa4k
-	./tests/dsm/extract_dsm_import_gpg_auto.tcl --p256
-	./tests/dsm/extract_dsm_import_gpg_auto.tcl --p384
-	./tests/dsm/extract_dsm_import_gpg_auto.tcl --p521
-	./tests/dsm/extract_dsm_import_gpg_auto.tcl --cv25519
-	./tests/dsm/sq_roundtrips.sh --rsa2k
-	./tests/dsm/sq_roundtrips.sh --rsa3k
-	./tests/dsm/sq_roundtrips.sh --rsa4k
-	./tests/dsm/sq_roundtrips.sh --p256
-	./tests/dsm/sq_roundtrips.sh --p384
-	./tests/dsm/sq_roundtrips.sh --p521
-	./tests/dsm/sq_roundtrips.sh --cv25519
+	./tests/dsm/key_expiration.sh -c rsa2k
```

Comment:
To reviewer: This `-c` option is only for the test script, it is not related to the objective of this PR. I did this because it is neater to use `getopts` in bash (see the test script common.sh)